src/v/storage/tests/segment_size_jitter_test.cc

```diff
@@ -19,12 +19,12 @@ SEASTAR_THREAD_TEST_CASE(test_segment_size_jitter_calculation) {
       100_MiB,
       storage::debug_sanitize_files::yes);
 
+    constexpr auto jitter = storage::jitter_percents(5);
```

Comment:
umm, actually cfg is not used anymore, its like we are testing if random_jitter works :-D, let me fix that.